src/pages/InformationSheetContent/InformationSheetContent.js

```diff
@@ -25,14 +25,12 @@ const InformationSheetContent = ({ crop, modalData }) => {
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
 
-  const section = window.location.href.includes('species-selector') ? 'selector' : 'explorer';
-
   // redux vars
   const regionIdRedux = useSelector((stateRedux) => stateRedux.mapData.regionId);
   const stateIdRedux = useSelector((stateRedux) => stateRedux.mapData.stateId);
   const apiBaseUrlRedux = useSelector((stateRedux) => stateRedux.sharedData.apiBaseUrl);
   const filterStateRedux = useSelector((stateRedux) => stateRedux.filterData);
-  const { zone } = filterStateRedux[section];
+  const { zone } = filterStateRedux.filters;
```

Comment:
does this exist in filterstateRedux.filters?

Reply:
and should it be region instead of zone?

Reply:
yes i updated the filterStateRedux to have a filters object to arrange things so the cropSearch was still located one step above the filters (previously selector and explorer) object as it was before.  Otherwise we would be adding the cropSearch value to the filters values and i thought that could cause confusion.  

and yes it should be region.